spec/features/tableau_de_bord_spec.rb

```diff
@@ -4,46 +4,119 @@
+        expect(page).to have_content(projet.agent_instructeur.fullname)
+        #TODO Theme
+        expect(page).to have_content(projet.agent_operateur.intervenant.raison_sociale)
+        expect(page).to have_content(projet.agent_operateur.fullname)
+        expect(page).to have_content(I18n.t("projets.statut.en_cours_d_instruction"))
+        #TODO Update Status At
       end
+    end
 
-      click_link projet.demandeur_principal.fullname
+    scenario "je peux accéder à un dossier à partir du tableau de bord" do
+      visit dossiers_path
+      click_link projet.numero_plateforme
       expect(page.current_path).to eq(dossier_path(projet))
       expect(page).to have_content(projet.demandeur_principal.fullname)
     end
+
+    scenario "je ne peux pas accéder au dossier Opal" do
+      visit dossiers_path
+      expect(page).not_to have_link(projet.opal_numero, :href=>dossier_opal_url(projet.opal_numero))
+    end
   end
 
   context "en tant qu'instructeur" do
-    let(:agent) { create :agent, intervenant: instructeur }
+    let(:current_agent) { agent_instructeur }
 
     # Attention les instructeurs voient tous les dossiers ! En attente de la règle métier
-    scenario "je peux accéder à un dossier à partir du tableau de bord" do
+    scenario "j'ai accès au tableau de bord avec les informations disponibles pour les instructeurs" do
       visit dossiers_path
       within "#projet_#{projet.id}" do
-        expect(page).to have_content(I18n.t("projets.statut.prospect"))
+        expect(page).to     have_content(projet.numero_plateforme)
+        expect(page).to     have_content(projet.opal_numero)
+        expect(page).to     have_content(projet.demandeur_principal.fullname)
+        expect(page).not_to have_content(projet.adresse.region)
+        expect(page).not_to have_css('td#departement')
+        expect(page).to     have_content(projet.adresse.ville)
+        expect(page).to     have_content(projet.agent_instructeur.intervenant.raison_sociale)
+        expect(page).to     have_content(projet.agent_instructeur.fullname)
+        #TODO Themes
+        expect(page).to     have_content(projet.agent_operateur.intervenant.raison_sociale)
+        expect(page).to     have_content(projet.agent_operateur.fullname)
+        expect(page).to     have_content(I18n.t("projets.statut.en_cours_d_instruction"))
+        #TODO Update Status At
       end
+    end
 
-      click_link projet.demandeur_principal.fullname
+    scenario "je peux accéder à un dossier à partir du tableau de bord" do
+      visit dossiers_path
+      click_link projet.numero_plateforme
       expect(page.current_path).to eq(dossier_path(projet))
       expect(page).to have_content(projet.demandeur_principal.fullname)
     end
+
+    scenario "je peux accéder au dossier Opal" do
+      visit dossiers_path
+      expect(page).to have_link(projet.opal_numero, :href=>dossier_opal_url(projet.opal_numero))
```

Comment:
Dans ce projet on utilise plutôt la syntaxe _JSON-like_ de Ruby pour les hash :

```ruby
expect(page).to have_link(projet.opal_numero, href: dossier_opal_url(projet.opal_numero))
```

http://breakthebit.org/post/8453341914/ruby-19-and-the-new-hash-syntax